### ISSUE TITLE
Fix conversion of placefolder on method completion.

### ISSUE THREAD

**Issue by mitsuse** [mitsuse/neocomplete-kitten#2]
For example, the plugin generates a completion candidate as follow:

``` swift
override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {^@<`1:code`>^@}
```

**Comment by mitsuse** [comment]
`^@` is represented in the json of a responsse from SourceKitten as follow:

``` json
[
  {
    ...
    "sourcetext" : "func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {\n<#code#>\n}",
    ...
  }
]
```

**Comment by mitsuse** [comment]
Fixed in a78cbb6.
